apis/feedapi/feedapi.yaml

```diff
@@ -0,0 +1,122 @@
+swagger: '2.0'
+info:
+  version: '1.0'
+  title: User Feed  API(s)
+  description: >-
+    - The User Feed API will notify user on various alerts like migration.
```

Comment:
1. `The User feeds is intended to store all alerts that a user could be presented with. This includes tenant migration.`
2. Remove the Note below.

Reply:
done